### PR TITLE
Docs: align Pootle FS statuses to current implementation

### PR DESCRIPTION
**Pull request by dwaynebailey**

### CODE REVIEW

File: docs/features/pootle_fs_statuses.rst [hunk 1]
```diff
@@ -8,8 +8,10 @@ these it is able to determine what to do to resolve the scenario.
 
 There are two groups of statuses:
 
-* Unstaged - these need to be resolved by the user and will become staged.
-* Staged - staged and ready to be `sync`'ed.
+* Unstaged - these need to be resolved by the user, once resolved they become
+  staged.  To resolve the user will use the :djadmin:`resolve`, :djadmin:`rm`
+  and djadmin:`add` commands.
```
Comment (unho) [inline docs/features/pootle_fs_statuses.rst]: Typo, it should be `:djadmin:`

File: docs/features/pootle_fs_statuses.rst [hunk 2]
```diff
@@ -21,38 +23,37 @@ to resolve them:
...
-  the Pootle store.
+  use either :option:`fetch --force` or :option:`add --force` depending on
+  whether you want to keep the filesystem file or the Pootle store.
 
 ``pootle_untracked``
   A new store has been added in Pootle but does not have any matching file on
   the filesystem. You can use either :djadmin:`add` to create the file on the
   filesystem and push the translations on the store to it, or alternatively use
-  :djadmin:`rm` to stage the store for removal.
+  :option:`rm --force` to stage the store for removal.
```
Comment (unho) [inline docs/features/pootle_fs_statuses.rst]: This should be `:command:`

File: docs/features/pootle_fs_statuses.rst [hunk 3]
```diff
@@ -21,38 +23,37 @@ to resolve them:
...
 
 ``fs_untracked``
   A new file has been added in the filesystem but does not have any matching
-  store in Pootle. You can use either :djadmin:`fetch` to pull the file into
-  Pootle or alternatively use :djadmin:`rm` to stage the file for removal.
+  store in Pootle. You can use either :djadmin:`add` to pull the file into
+  Pootle or alternatively use :option:`rm --force` to stage the file for
```
Comment (unho) [inline docs/features/pootle_fs_statuses.rst]: `:command:` too

File: docs/features/pootle_fs_statuses.rst [hunk 4]
```diff
@@ -21,38 +23,37 @@ to resolve them:
...
+  removal.
 
 ``pootle_removed``
-  A tracked store has been removed. Either use :option:`fetch --force
-  <fetch --force>` to restore the filesystem version, or use :djadmin:`rm` to
-  stage for removal from filesystem.
+  A tracked store has been removed. Either use :option:`add --force` to restore
```
Comment (unho) [inline docs/features/pootle_fs_statuses.rst]: And here `:command:` as well

File: docs/features/pootle_fs_statuses.rst [hunk 5]
```diff
@@ -21,38 +23,37 @@ to resolve them:
...
+  the filesystem version, or use :djadmin:`rm` to stage for removal from
+  filesystem.
 
 ``fs_removed``
-  A tracked file has been removed from the filesystem. Either use :option:`add
-  --force <add --force>` to restore the Pootle version, or use :djadmin:`rm` to
+  A tracked file has been removed from the filesystem. Either use
+  :option:`add --force` to restore the Pootle version, or use :djadmin:`rm` to
```
Comment (unho) [inline docs/features/pootle_fs_statuses.rst]: And `:command` as well

File: docs/features/pootle_fs_statuses.rst [hunk 6]
```diff
@@ -21,38 +23,37 @@ to resolve them:
 
 ``conflict``
   Both the Pootle store and the filesystem file have changed. To resolve this
-  conflict use :djadmin:`merge` to merge the two and manage conflict
-  resolution. Or if you wish to discard one or the other use :option:`fetch
-  --force <fetch --force>` or :option:`add --force <add --force>` to keep the
-  filesystem version or Pootle version respectively.
+  conflict use :djadmin:`resolve` to merge the two and manage conflict
+  resolution. Or if you wish to discard one or the other use :option:`resolve
+  --overwrite`.
```
Comment (unho) [inline docs/features/pootle_fs_statuses.rst]: I think we need clarity on how handle each of both scenarios:

> Use :command:`resolve --overwrite --pootle-wins` if you want to keep Pootle version, or :command:`resolve --overwrite` if you want to keep the filesystem version.

**Comment by unho** [comment]
lgtm once all concerns are addressed.

File: docs/features/pootle_fs_statuses.rst (hunk 2 above)
Comment (dwaynebailey) [reply docs/features/pootle_fs_statuses.rst]: @unho no it should not, it should be `:option:`

File: docs/features/pootle_fs_statuses.rst (hunk 6 above)
Comment (dwaynebailey) [reply docs/features/pootle_fs_statuses.rst]: We already have that elsewhere so I didn't really want to repeat that here again, so I've tried to keep it abstract.

**Pull request merged**